package.json

```diff
@@ -2,8 +2,9 @@
     "private": true,
     "devDependencies": {
         "@types/node": "16.11.54",
-        "inline-fixtures": "1.1.0",
-        "typescript": "4.8.2"
+        "inline-fixtures": "https://registry.yarnpkg.com/inline-fixtures/-/inline-fixtures-1.1.0.tgz#5f9edf644ac02a1322012dbbb9f0a4739831d662",
+        "typescript": "4.8.2",
+        "json-stable-stringify": "1.0.1"
```

Comment:
What is this used for? Did this reproduce the problem?

Reply:
I did now, https://github.com/aspect-build/rules_js/issues/423#issuecomment-1233130904

Reply:
Will put this back to DRAFT and include the fix for name issue in #423

Reply:
It would be nice to have a more explicit test. Or at least a comment with a link to the issue so no one sees this "unused dependency" and deletes it, or upgrades it to a version no longer reproducing the problem etc.

Maybe we could have an e2e test for tests like this? Somewhere to put package/lock related tests instead of putting them in this root package.json?

Reply:
I have a spot already. Will hoist them to npm/private/test/package.json